pytesmo/metrics.py

```diff
@@ -342,6 +342,159 @@ def tcol_snr(x, y, z, ref_ind=0):
     return snr, np.sqrt(err_var) * beta, beta
 
 
+def EC(data, correlated=None, err_cov=None, abs_est=True):
```

Comment:
Could you please make this lowercase `ec` or maybe `ecol` to be consistent with `tcol` in the other functions.